packages/stardog-graphql-language-server/src/GraphQlLanguageServer.ts

```diff
@@ -239,6 +379,28 @@ export class GraphQlLanguageServer extends AbstractLanguageServer<
       tokenVectorForContentAssist
     );
 
+    if (tokenVectorForContentAssist.length > 0) {
+      // These candidates are relevant only after at least one token.
+      this.addStardogSpecificDirectivesToCompletionCandidates(candidates);
+
+      const firstNonWhitespaceBeforeCursor = document
+        .getText()
+        .slice(0, cursorOffset)
+        .trim()
+        .slice(-1);
+
+      if (
+        firstNonWhitespaceBeforeCursor ===
+          stardogGraphQlTokenMap.LParen.PATTERN ||
+        firstNonWhitespaceBeforeCursor === stardogGraphQlTokenMap.Comma.PATTERN
+      ) {
+        this.addStardogSpecificArgumentsToCompletionCandidates(
+          candidates,
+          tokenVectorForContentAssist
+        );
+      }
+    }
```

Comment:
I am currently not getting the completion for arguments. Don't we also have to account for the indices of what is currently filled out so far for the potential argument, if I'm understanding what's happening here correctly? (ex: 't' = 1 space, 'ali' = 3 spaces).

![Screen Shot 2021-02-19 at 12 03 04 PM](https://user-images.githubusercontent.com/49697587/108536411-72258380-72aa-11eb-8592-5b605c1d43b8.png)

![Screen Shot 2021-02-19 at 12 09 16 PM](https://user-images.githubusercontent.com/49697587/108537183-4fe03580-72ab-11eb-876c-90ea294760a5.png)

Reply:
This should be fixed now.